### DISCUSSION TITLE
xPack Arm Embedded GCC 10.3.1-2.1 released

### DISCUSSION THREAD

**Discussion by ilg-ul** [stm32duino/Arduino_Core_STM32#1537] [Show and tell]
The latest Arm toolchain (arm-none-eabi-gcc) release 10.3-2021.10 from October 21, 2021 is now available as xPack 10.3.1-2.1:

- <https://xpack.github.io/blog/2021/10/25/arm-none-eabi-gcc-v10-3-1-2-1-released/>

It would be great to update Arduino to use it.

**Comment by fpistm** [comment]
Hi @ilg-ul 
Thanks for the notification. I've saw the release. 
It will probably be part of the next release.

**Reply by ilg-ul** [reply]
Ok. Please test it and let me know if it ok.

**Comment by ilg-ul** [comment]
BTW, did you try to integrate the toolchain with the new Arduino IDE 2?

**Reply by fpistm** [reply]
I've tried several months ago to install the STM32 core and it works as expected.

**Comment by fpistm** [comment]
New release use it: https://github.com/stm32duino/Arduino_Core_STM32/releases/tag/2.2.0
